kie-server-parent/kie-server-services/kie-server-services-task-assigning/kie-server-services-task-assigning-core/src/main/java/org/kie/server/services/taskassigning/core/model/User.java

```diff
@@ -53,12 +55,20 @@ public void setGroups(Set<Group> groups) {
         this.groups = groups;
     }
 
-    public Set<TypedLabel> getTypedLabels() {
-        return typedLabels;
+    public Map<String, Set<Object>> getLabelValues() {
+        return labelValues;
     }
 
-    public void setTypedLabels(Set<TypedLabel> typedLabels) {
-        this.typedLabels = typedLabels;
+    public void setLabelValues(Map<String, Set<Object>> labelValues) {
+        this.labelValues = labelValues;
+    }
```

Comment:
Personally, I would keep only `setLabelValues(labelName, values)`. If you have to keep this method as well, consider at least defensively copying the `Map`.

Reply:
the setLabelValues(Map<String, Set<Object>> labelValues) was mainly defined in case this is something that Solution cloners, or any other internal OptaPlanner mechanims use. ¿?